django/contrib/postgres/indexes.py

```diff
@@ -223,3 +223,10 @@ def get_with_params(self):
         if self.fillfactor is not None:
             with_params.append('fillfactor = %d' % self.fillfactor)
         return with_params
+
+
+class OpClass(Func):
+    template = '%(expressions)s %(opclass)s'
+
+    def __init__(self, *expressions, opclass):
```

Comment:
@hannseman As far as I'm aware an operator class should be defined for each expression. Currently we allow for multiple expression but they are ignored later, we should accept only a single expression (as in `OrderBy()`). I pushed edits, what do you think?

Reply:
You're right, it should only accept a single expression. BTW when adding `OpClass` I had some thoughts about if the `opclass` argument should be named `name` instead, forgot to bring that up. What do you think?

Reply:
Agreed